include/villas/kernel/vfio.hpp

```diff
@@ -0,0 +1,34 @@
+/** VFIO Kernel module loader
+ *
+ * Author: Pascal Bauer <pascal.bauer@rwth-aachen.de>
+ * SPDX-FileCopyrightText: 2017 Institute for Automation of Complex Power
+ * Systems, EONERC SPDX-License-Identifier: Apache-2.0
+ *********************************************************************************/
+
+//TODO: Move to cmake args
+//#define FPGA_PLATFORM
```

Comment:
```suggestion
// TODO: Move to cmake args
// #define FPGA_PLATFORM
```

Please also consider moving this into `config.hpp.in` using a `#cmakedefine`

Reply:
I think this is fine for now - as long as there is no regression when not setting this.

Reply:
As my comment suggests, a cmake define would be much cleaner. I will look up the docs.